### PR TITLE
fix: test after getting focus

### PR DESCRIPTION
**Pull request by subirjolly**
Closes #

### CODE REVIEW

**Review by Asalem1 — COMMENTED** [review]
Not related to this PR but if we're already pushing up test changes, I figured we could probably unskip this test and remove the comment since it looks like the issue was closed:

https://github.com/influxdata/ui/pull/1198/files#diff-42baf636a2d28b8896d300a8355261b996ce9fa35f2bff1788421b9bae0782b9R356

**Comment by hercules-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?vars%5Blighthouse_score_branch%5D=1198)

**Review by Asalem1 — APPROVED** [review]

File: cypress/e2e/shared/tasks.test.ts [hunk 1]
```diff
@@ -50,15 +50,15 @@ from(bucket: "${name}"{rightarrow}
       cy.get('.cf-overlay--dismiss').click()
     })
   })
-  // this test is broken due to a failure on the post route
-  it.skip('can create a task using http.post', () => {
+
+  it('can create a task using http.post', () => {
     const taskName = 'Task'
     createFirstTask(taskName, () => {
-      return `import "http{rightarrow}
```
Comment (Asalem1) [inline cypress/e2e/shared/tasks.test.ts:57]: i think this might fail due to formatting

File: cypress/support/commands.ts [hunk 2]
```diff
@@ -558,6 +558,19 @@ export const getByTestID = (
   return cy.get(`[data-testid="${dataTest}"]`, options)
 }
 
+export const getByTestIDAndSetInputValue = (
```
Comment (genehynson) [inline cypress/support/commands.ts:561]: good call on moving this to this file!

File: cypress/e2e/shared/tasks.test.ts [hunk 3]
```diff
@@ -628,10 +622,15 @@ http.post(
...
 
       // Fill Task Form
-      getByTestIdAndSetInputValue('task-form-name', task.name)
-      getByTestIdAndSetInputValue('task-form-schedule-input', task.every)
-      getByTestIdAndSetInputValue('task-form-offset-input', task.offset)
+      // focused() waits for monoco editor to get input focus
+      // If this isn't present then cypress shifts focus on elements
+      // making it seem randomly jumping to elements
+      cy.focused()
+
```
Comment (genehynson) [inline cypress/e2e/shared/tasks.test.ts:629]: nice find

**Review by genehynson — APPROVED** [review]
thanks for unskipping those other tests!!!

**Pull request merged**